### ISSUE TITLE
Error: Packet structure error

### ISSUE THREAD

**Issue by imnetcat** [metarhia/metacom#237]
```
12:32:49  W1   error   172.20.0.1  GET     /       500     Error: Packet structure error
api_1      |   Channel.message (/usr/server/node_modules/impress/node_modules/metacom/lib/channel.js:189:21)
api_1      |   WebSocket.<anonymous> (/usr/server/node_modules/impress/node_modules/metacom/lib/server.js:61:17)
api_1      |   WebSocket.emit (node:events:394:28)
api_1      |   Receiver.receiverOnMessage (/usr/server/node_modules/impress/node_modules/ws/lib/websocket.js:1009:20)
api_1      |   Receiver.emit (node:events:394:28)
api_1      |   Receiver.dataMessage (/usr/server/node_modules/impress/node_modules/ws/lib/receiver.js:522:14)
api_1      |   Receiver.getData (/usr/server/node_modules/impress/node_modules/ws/lib/receiver.js:440:17)
api_1      |   Receiver.startLoop (/usr/server/node_modules/impress/node_modules/ws/lib/receiver.js:148:22)
api_1      |   Receiver._write (/usr/server/node_modules/impress/node_modules/ws/lib/receiver.js:83:10)
api_1      |   writeOrBuffer (node:internal/streams/writable:389:12)
```

**Describe the bug**
The error above occurs randomly after starting the server, even without any requests to API
[Looks like this ](https://github.com/metarhia/metacom/issues/231)

**To Reproduce**
Start impress v2.0.1, metacom v2.6.1
the same error was also noticed on metacom ^v1.8.5

**Expected behavior**
No error occurred

**Screenshots**
In every 30-40 seconds this error occurred, even without request to the API 
![image](https://user-images.githubusercontent.com/37350375/132855197-c5e33e37-aa2d-4334-b8c0-ffff6aed0a09.png)

**Desktop:**
- OS: Alpine Linux node js docker image
- Node.js version v14.17.3

**Additional context**
It may cause by React, but I dont think so

**Comment by imnetcat** [comment]
Solved in v2.0.2

**Comment by tshemsedinov** [comment]
Duplicate: https://github.com/metarhia/metacom/issues/231
